docs/partners/become-a-tech-partner.md

```diff
@@ -106,13 +106,13 @@ At a minimum, an integration consists in:
 
 ## 6. Provide our Team with Product Access
 
-As an automation platform, it shouldn’t surprise you that we’ve also automated our integration testing and deployment. We do nightly tests of each integration in order to confirm functionality and, in order to do this, we require access to your product and/or APIs. Preferably, the instance will have all data related to the integration use cases, so they can be tested. E.g. if one of the use cases is ingesting incidents to Cortex XSOAR, the instance should have demo incidents. This access is not used for any other purpose.
+As an automation platform, it shouldn’t surprise you that we’ve also automated our integration testing and deployment. We do nightly tests of each integration in order to confirm functionality and, in order to do this, we request access to your product and/or APIs. Preferably, the instance will have all data related to the integration use cases, so they can be tested. E.g. if one of the use cases is ingesting incidents to Cortex XSOAR, the instance should have demo incidents. This access is not used for any other purpose.
 
-Before submitting the integration, please work with the Palo Alto Networks BD Team to share access
+Before submitting the integration, please work with the Palo Alto Networks BD Team to share access.
```

Comment:
Can we completely remove this part? we. don't enforce or check it. anymore at all.

Reply:
I went back and forth about it, will remove.